codestodians/komlie-frontend
# [Feature] Activate profile image upload

#### What does this PR do?
- Allows for the upload of a profile image when updating a user's details

#### Description of Task to be completed?
- Create `uploadImage` util

#### How should this be manually tested?
- Check out into the `feature/add-profile-picture-field` on the api, run the migration and start the app
- Check out into the `feature/activate-profile-image-upload` on the frontend, run `npm install` and then start the app.
- Select any image and upload

#### Any background context you want to provide?
- There was no provision for adding an image to a user profile, so the PR open on the api took care of that before this was implemented.

#### What are the relevant pivotal tracker stories?
- [#167160649](https://www.pivotaltracker.com/story/show/167160649)
